mne/viz/backends/_ipyvolume.py

```diff
@@ -0,0 +1,500 @@
+
+        Parameters
+        ----------
+        surface: surface object
+            The mesh to use as support for contour.
+        scalars: ndarray, shape (n_vertices)
+            The scalar valued associated to the vertices.
+        contours: int | list
+             Specifying a list of values will only give the requested contours.
+        line_width: float
+            The width of the lines.
+        opacity: float
+            The opacity of the contour.
+        vmin: float | None
+            vmin is used to scale the colormap.
+            If None, the min of the data will be used
+        vmax: float | None
+            vmax is used to scale the colormap.
+            If None, the max of the data will be used
+        colormap:
+            The colormap to use.
+        """
+        from matplotlib import cm
+        from matplotlib.colors import ListedColormap
+        from ipyvolume.pylab import plot
+
+        vertices = surface['rr']
+        tris = surface['tris']
+
+        if colormap is None:
+            cmap = cm.get_cmap('coolwarm')
+        else:
+            cmap = ListedColormap(colormap / 255.0)
+
+        if isinstance(contours, int):
+            levels = np.linspace(vmin, vmax, num=contours)
+        else:
+            levels = np.array(contours)
+
+        if scalars is not None:
+            if vmin is None:
+                vmin = min(scalars)
+            if vmax is None:
+                vmax = max(scalars)
+            nscalars = (scalars - vmin) / (vmax - vmin)
+            color = cmap(nscalars)
+        else:
+            color = np.append(color, opacity)
+
+        verts, faces, _, _ = _isoline(vertices=vertices, tris=tris,
+                                      vertex_data=scalars,
+                                      levels=levels)
+
+        x = verts[:, 0]
+        y = verts[:, 1]
+        z = verts[:, 2]
+        plot(x, y, z, color=color)
+
+    def surface(self, surface, color=None, opacity=1.0,
+                vmin=None, vmax=None, colormap=None, scalars=None,
+                backface_culling=False):
+        """Add a surface in the scene.
+
+        Parameters
+        ----------
+        surface: surface object
+            The information describing the surface.
+        color: tuple
+            The color of the surface: (red, green, blue).
+        opacity: float
+            The opacity of the surface.
+        vmin: float | None
+            vmin is used to scale the colormap.
+            If None, the min of the data will be used
+        vmax: float | None
+            vmax is used to scale the colormap.
+            If None, the max of the data will be used
+        colormap:
+            The colormap to use.
+        scalars: ndarray, shape (n_vertices,)
+            The scalar valued associated to the vertices.
+        backface_culling: bool
+            If True, enable backface culling on the surface,
+            no support yet.
+        """
```

Comment:
Rebase, remove these and use the nice new decorator

Reply:
Right on it!